docs/modules/servers/pages/distributed/configure/ssl.adoc

```diff
@@ -21,13 +21,33 @@ for TLS 1.0 which, as a result, is sometimes referred to as SSL 3.1.
 You need to add a block in the corresponding configuration file (smtpserver.xml, pop3server.xml, imapserver.xml,..)
 
 ....
-<tls socketTLS="false" startTLS="false">
+<tls socketTLS="false" startTLS="true">
   <keystore>file://conf/keystore</keystore>
+  <keystoreType>PKCS12</keystoreType>
   <secret>yoursecret</secret>
   <provider>org.bouncycastle.jce.provider.BouncyCastleProvider</provider>
 </tls>
 ....
 
+Alternatively TLS keys can be supplied via PEM files:
+
+....
+<tls socketTLS="true" startTLS="false">
+  <privateKey>file://conf/private.key</privateKey>
+  <certificates>file://conf/certs.self-signed.csr</certificates>
```

Comment:
Often certificates can be provided as a single file and/or as a directory. Would it be possible also to add the complementary directory option?

Reply:
IMO could be. Changes would be welcome, feel free to propose an exact format and to open a PR to complete this one .